commands/operator-sdk/cmd/new.go

```diff
@@ -72,22 +62,17 @@ const (
 )
 
 func newFunc(cmd *cobra.Command, args []string) {
-	if len(args) != 1 {
-		cmdError.ExitWithError(cmdError.ExitBadArgs, fmt.Errorf("new command needs 1 argument."))
-	}
 	parse(args)
 	mustBeNewProject()
-	verifyFlags()
-	g := generator.NewGenerator(apiVersion, kind, projectName, repoPath())
-	err := g.Render()
-	if err != nil {
-		cmdError.ExitWithError(cmdError.ExitError, fmt.Errorf("failed to create project %v: %v", projectName, err))
-	}
+	// TODO: codegen
 	pullDep()
 	initGit()
 }
 
 func parse(args []string) {
+	if len(args) != 1 {
+		cmdError.ExitWithError(cmdError.ExitBadArgs, fmt.Errorf("new command needs 1 argument"))
```

Comment:
Possibly stop using ExitWithError and simplify the above to:
```
logrus.Fatal("new command needs 1 argument")
```

Reply:
yeah, let's do `log.Fatal()` instead of `logrus` because we don't really need the extra dependency.